src/bootstrap/src/core/builder.rs

```diff
@@ -829,6 +829,7 @@ impl<'a> Builder<'a> {
                 test::Bootstrap,
                 // Run run-make last, since these won't pass without make on Windows
```

Comment:
`run-make`  tests are not run last anymore.

Reply:
Oh, sorry.

Reply:
Note that my comment is totally random. I just noticed that the comment was not *totally* accurate anymore.
I have no idea if you need to move run-make in last position or if you need to change the comment.